### PR TITLE
docs: add missing step to test dapp docs

### PR DESCRIPTION
**Pull request by RichAyotte**
## Summary
Add missing steps to test dapp docs

### Testing
The steps in these updated instructions were followed exactly. I also deviated by skipping a step or running the commands in the wrong directories to assure that all the steps are needed and in the specified order and in the proper directories.

### CODE REVIEW

File: apps/taquito-test-dapp/README.md [hunk 1]
```diff
@@ -4,13 +4,18 @@
...
-3. Install all dependencies:
-    `npm clean-install`
-4. Start development server:
-    `npm run dev`
-5. Open http://localhost:3030 in your browser to see a sample application.
-6. Start production server:
-    `npm run build && npm run preview`
-7. Open http://localhost:4173 in your browser to see a preview application.
+1. Change your current working directory to the newly cloned repository.
+1. Install dependencies: `npm i`
```
Comment (egarson) [inline apps/taquito-test-dapp/README.md:10]: We don't use `install`, but rather `clean-install` so it respects package.json and we get deterministic builds. Also, prefer using the full name of the command `clean-install` for documentation purposes, programmers that need `ci` will find it :wink:

File: apps/taquito-test-dapp/README.md [hunk 2]
```diff
@@ -4,13 +4,18 @@
...
+1. Change your current working directory to `apps/taquito-test-dapp`.
+1. Start development or production server as shown below.
+
+#### Start development server:
+1. `npm run dev`
+1. Open http://localhost:3030 in your browser to see a sample application.
+
+#### Start production server:
+1. `npm run build && npm run preview`
+1. Open http://localhost:4173 in your browser to see a preview application.
```
Comment (egarson) [inline apps/taquito-test-dapp/README.md:21]: And thank you for the above: it's a nice improvement!

**Review by egarson — CHANGES_REQUESTED** [review]
Just one small change and it's good to go

File: apps/taquito-test-dapp/README.md (hunk 2 above)
Comment (RichAyotte) [reply apps/taquito-test-dapp/README.md]: Thanks for the quick review!

**Comment by egarson** [comment]
LGTM :+1:

**Pull request merged**